sdk/attestation/attestation/src/attestationClient.ts

```diff
@@ -138,29 +278,34 @@ export class AttestationClient {
   }
 
   /**
-   * 
+   * Returns the OpenID Metadata discovery document for the attestation service instance.
    * @param options Client operation options.
    * @returns The OpenID metadata discovery document for the attestation service.
    */
   public async getOpenIdMetadata(options: AttestationClientOperationOptions = {}) : Promise<any>
   {
-    const { span, updatedOptions} = createSpan("AttestationClient-getAttestationSigners", options);
+    const { span, updatedOptions} = createSpan("AttestationClient-getOpenIdMetadata", options);
     try {
       return await (await this._client.metadataConfiguration.get(updatedOptions)).body;
     } catch (e) {
       span.setStatus({ code: SpanStatusCode.ERROR, message: e.message});
     } finally {
       span.end();
     }
-
   }
   
 
   private _client: GeneratedClient;
 
   instanceUrl: string;
 
+  /**
+   * @ignore
```

Comment:
we do not use ignore. Please follow these guidelines: https://github.com/Azure/azure-sdk-for-js/issues/13459

Reply:
I spent a couple of hours trying to find anything that would work, this was part of that experiment. Basically there doesn't seem to be a way of saying "Ignore this type" :(.